examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py

```diff
@@ -91,7 +91,9 @@
 ρ = 10  # ADMM penalty parameter
 σ = density * 0.26  # denoiser sigma
 
-f = SVMBIRWeightedSquaredL2Loss(y=y, A=A, W=Diagonal(weights), scale=0.5)
+f = SVMBIRWeightedSquaredL2Loss(
+    y=y, A=A, W=Diagonal(weights), scale=0.5, prox_kwargs={"maxiter": 5}
```

Comment:
Is it really desirable that we just drop the default stopping tolerance here?

Reply:
I think I would choose the tolerance to be 0 (or very) low so that always 5 iterations are being performed (unless there is really almost no change)

Reply:
-> a640fa7